docs/integrations/keras_integration.rst

```diff
@@ -0,0 +1,597 @@
+
+Neptune-Keras Integration
+*************************
+
+What will you get with this integration
+=======================================
+
+In this tutorial, you'll learn how to integrate Neptune into your Keras application, and at the end you'll be able to efficiently track and monitor model training, metrics, images and even datasets.
+
+Video [1min screencast]
+
+|Keras| is a high level interactive API that can be used to prototype, build and train neural networks. \
```

Comment:
This what is Keras seciton is a bit too long -> could you please make it shorter (1-2 sentences tops).